components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/internal/OAuthComponentServiceHolder.java

```diff
@@ -445,4 +447,14 @@ public void setCorsManagementService(CORSManagementService corsManagementService
 
         this.corsManagementService = corsManagementService;
     }
+
+    public AuthorizedAPIManagementService getAuthorizedAPIManagementService() {
```

Comment:
Add method comments to the public methods

Reply:
Added